app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/check_all_answers.scala.html

```diff
@@ -91,14 +93,33 @@
         case _ => Html("")
     }
 }
-@isAgent = @{ request.userType.contains(UserType.Agent) }
+@isAnAgent = @{ request.userType.contains(UserType.Agent) }
+
+@accountNamePrefix = @{
+    fillingOutReturn.draftReturn.representativeType match {
+        case Some(PersonalRepresentative) =>
+            s"${messages("account.personalRep.prefix")} ${fillingOutReturn.subscribedDetails.makeAccountName()}"
```

Comment:
the name for cap/pr should come from the representee answers

Reply:
this same logic is repeated in a number of places - it could be factored out somewhere perhaps